wfsim/load_resource.py

```diff
@@ -101,48 +101,80 @@ def file_config(config):
 
         return files
 
+    @staticmethod
+    def get_file_path(base, fname):
+        """Find the full path to the resource file
+        Try 4 methods in the following order
+        1. The base is not url, return base + name
+        2. If ntauxfiles (straxauxfiles) is installed, return will be package dir + name
+           pip install won't work, try python setup.py in the packages
+        3. Download using straxen mongo downloader from mongo database,
+           return the cached file path + hashed name
+        4. Download using straxen get_resource from the url (github raw)
+           simply return base + name
+
+            Be carefull with the third and forth options, straxen creates
+            cache files that might not be updated with the latest mongodb entry or github commit.
+
+        """
+        if base.startswith('/'):
+            log.warning(f"Using local file {fname} for a resource. "
+                        f"Do not set this as a default or TravisCI tests will break")
+            return osp.join(base, fname)
+
+        try:
+            # You might want to use this, for example if you are a developer
+            log.warning(f"Using the private repo to load {fname} locally")
+            import ntauxfiles  # private package for XENONnT
+            fpath = ntauxfiles.get_abspath(fname)
+            log.info(f"Loading {fname} is successfully from {fpath}")
+            return fpath
+        except (ModuleNotFoundError, ImportError, FileNotFoundError):
+            log.info(f"ntauxfiles is not installed or does not have {fname}")
+
+        try:
+            # You might want to use this, for example if you are a developer
+            log.warning(f"Using the public repo to load {fname} locally")
+            import straxauxfiles  # public package for XENON1T
+            fpath = ntauxfiles.get_abspath(fname)
```

Comment:
Also you can use `fname in straxauxfiles.list_aux_files()`  and `fname in ntauxfiles.list_private_files()` to thereby check if the file is in either and remove all this `try: except` logic.